Toolset/libraries/revidelibrary.8.livecodescript

```diff
@@ -10931,7 +10935,7 @@ private command __tryToDoScriptInObject pObject, pToDo, @rValidScript
    put the result into tError
    if tError is empty then
       try
-         send "-- execute in object context" & return & tDoScript to pObject
+         send tDoScript to pObject
```

Comment:
Hmm... it sounds like we want a way to call domess directly without trying to pre-parse parameters